File: src/StealME.Client/src/stealme/client/networking/LocationMessage.java
package stealme.client.networking;

import org.msgpack.annotation.Message;

@Message
public class LocationMessage {
	 public String Latitude;
     public String Longitude;
     public String MNC;
     public String MCC;
     public String LAC;
     public String CellID;
     public String LatestRssi;
}

File: src/StealME.Server/StealME.Server.Messaging/Messages/LocationMessage.cs
namespace StealME.Server.Messaging.Messages
{
    public class LocationMessage
    {
        public double Latitude;
        public double Longitude;
        public int MNC;
        public int MCC;
        public int LAC;
        public int CellID;
        public int LatestRssi;
    }
}
